src/components/common/nav-menu/index.js

```diff
@@ -11,12 +11,8 @@ const NavMenu = () => (
         </Navbar.Brand>
       </Navbar.Header>
       <Nav pullRight>
-        <NavItem>
-          Register
-        </NavItem>
-        <NavItem>
-          Login
-        </NavItem>
+        <NavItem href="/sign-up">Sign Up</NavItem>
```

Comment:
Create a constants file where all your routes are defined. Then, import and use the route in the "href" properties here.